Inference - add MiniMaxAI/MiniMax-M2.5 (#2186)
wandb/docs: inference/models.mdx

| Meta Llama 3.1 70B                             | `meta-llama/Llama-3.1-70B-Instruct`         | Text         | 128K           | 70B (Total)               | Efficient conversational model optimized for responsive multilingual chatbot interactions.                                                   |
| Meta Llama 3.1 8B                              | `meta-llama/Llama-3.1-8B-Instruct`          | Text         | 128K           | 8B (Total)                | Efficient conversational model optimized for responsive multilingual chatbot interactions.                                                   |
| Microsoft Phi 4 Mini 3.8B                      | `microsoft/Phi-4-mini-instruct`             | Text         | 128K           | 3.8B (Total)              | Compact, efficient model ideal for fast responses in resource-constrained environments.                                                      |

> | MiniMax M2.5                                   | `MiniMaxAI/MiniMax-M2.5`                    | Text         | 197K           | 10B-230B (Active-Total)   | MiniMax M2.5 is an MoE model with a highly sparse architecture allowing for high-throughput and low-latency with strong coding capabilities. |

Reviewer: ```suggestion
| MiniMax M2.5                                   | `MiniMaxAI/MiniMax-M2.5`                    | Text         | 197K           | 10B-230B (Active-Total)   | MoE model with a highly sparse architecture designed for high-throughput and low latency with strong coding capabilities. |
```
Truncated a bit to align with the other entries and avoid the ambiguous indefinite article based on how you say the acronym.
